libraries/SITL/SITL.cpp

```diff
@@ -697,14 +697,94 @@ const AP_Param::GroupInfo SIM::var_ins[] = {
     AP_GROUPINFO("GYR3_BIAS",     33, SIM, gyro_bias[2], 0),
 #endif
 
+#if INS_MAX_INSTANCES > 3
```

Comment:
let's change to using SIM::InsParm like we did for SIM::BaroParam, it would make SIM IMU parameters much saner

Reply:
It looks like that's going to be quite a complex change as well @tridge . Should I add to this PR? A separate PR might be a better way?

Reply:
yes, separate it fine, lets skip it for this PR